internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -769,6 +769,25 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 										}
 									}
 								}
+								// do we have a logging parameter in our rule?
+								if rules.([]interface{})[ruleCounter].(map[string]interface{})["logging"] != nil {
+									// make sure it has data
+									if rules.([]interface{})[ruleCounter].(map[string]interface{})["logging"].(map[string]interface{})["enabled"] != nil {
+										/*
+											enabled = true -> status = 'enabled'
+											enabled = flase -> status = 'disabled'
+											Also lets only add 'status' if it doesn't already exist
```

Comment:
if it doesn't exist, it shouldn't be here as that is the default and we don't want to send it.

Reply:
in some cases, `status` won't exist when `enabled` is set. This is the case for the latest clone we tried to do. What i was trying to say in my comment, and maybe i need to make this more clear, is only set status if it doesn't exist and enabled is set.